_posts/2020-08-31-faf-rename.markdown

```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title:  "FAF is dead, long live ABRT Analytics"
+date:   2019-03-28 18:48:44 +0100
+author: Miroslav Suchý
+visible: 1
+categories: FAF Analytics
+---
+ABRT team decided to rename [FAF](https://github.com/abrt/faf/) to new name: **ABRT Analytics**.
+
+Disclaimer: this blogpost was written in 2019-03-28. But I forgot to publish it. Lets do that now. :)\
+
+There were several reasons. FAF stands for Fedora Analysis Framework, but it is for a long time not bound only to Fedora. While we support deployment to Fedora and RHEL/CentOS only, it can analyze crashes from any system or container.
+
+FAF and ABRT do not sound to be related, and you have to read a lot of documentation to find the relation between two of them. From a marketing point of view it should be evident that they are tightly connected. ABRT daemon and client tools gather crashes and reports and FAF analyze them, aggregate them.
+
+We discussed this in the team and we had two favorites:
+
+  * ABRT Dashboard
+  * ABRT Analytics
+
+After some time, we have chosen **ABRT Analytics** - because it shows some analysis of reports.
+
+I [changed](https://github.com/abrt/faf/pull/759) all string FAF to ABRT Analytics in WebUI. We will start changing references in the documentation. There is **no** plan to rename GitHub projects or massively rename all variables and references in code as that would require big effort, and we instead focus on fixing bugs and implementing features.
```

Comment:
s/massively rename all variables and references in code/rename all variables and references in the code en masse/